Blog/gst-build-cross-build/cross-building-with-gstreamer.md

```diff
@@ -0,0 +1,320 @@
+
+Another build system named [cerbero](https://gitlab.freedesktop.org/gstreamer/cerbero), implemented a few years ago, provides a standalone solution to building GStreamer packages. This solution offers a wide range of options in addition to a proper sandbox to avoid system dependencies and to be able to prepare packages according to proper third party software dependencies for a given version. `cerbero` is written in Python and can build for the host machine like `gst-uninstalled` but also for various common targets depending on the host. Indeed a Linux regular desktop host will offer to cross-build GStreamer for x86(32/64bits) but also for archictecture such *ARM* and system such as *Microsoft Windows*. Despite a shell environment allowing artifact testing, `cerbero` is not really convenient for a day to day development related to GStreamer as a new plugin development or a bug fix as it is not easy to update to the last revision without loosing a current work, or to test another branch of GStreamer
+
+
+### The Rise of gst-build:
+
+In order to improve this situation,  [gst-build](https://gitlab.freedesktop.org/gstreamer/gst-build) was born .
+Taking advantage of the flexibility of the rising build system, [meson](https://mesonbuild.com/), `gst-build` has been implemented to replace `gst-uninstalled` and provide a quick and smooth environment to hack into GStreamer and its dependencies.
+
+
+### Autotools Is Dead, Long Live Meson:
+
+Since GStreamer 1.18, `meson` has been chosen as the only build system for the official GStreamer repositories. For its simplicity, speed and flexibility, `meson` replaces `autotools`, so it is also perfect for use with `gst-build`. Indeed `gst-build` is first and foremost a `meson` project including `GStreamer` sub-projects with options to enable/disable selected sub-projects.
+
+So lets take a look on how to get started with `gst-build`:
+
+
+### A first step using gst-build:
+
+####A few bits about it:
+
+`gst-build` is mainly a `meson.build` project. It reads .wrap files which are located in the `subprojects` folder to determine the elements of the project such as GStreamer or Glib. These subprojects use the `meson` build system. `gst-build` comes with the essential projects you need to start using GStreamer and build it almost without system dependencies. BY the way, GStreamer framework does not need system dependencies as gst-build bundles `libffi` or `glib`. but can also gather the dependencies using `pkg-config` from the system to build the GStreamer plugins such as flac, for example, which needs libflac to build.
```

Comment:
This is partially true. A lot of deps for `ext` plugins are not bundled with gst-build atm.

Reply:
I tried to rephrase a bit